apps/condo/domains/acquiring/schema/RegisterMultiPaymentService.js

```diff
@@ -479,6 +498,186 @@ const RegisterMultiPaymentService = new GQLCustomSchema('RegisterMultiPaymentSer
                 }
             },
         },
+        {
+            access: access.canRegisterMultiPayment,
+            schema: 'registerMultiPaymentForOneReceipt(data: RegisterMultiPaymentForOneReceiptInput!): RegisterMultiPaymentOutput',
+            resolver: async (parent, args, context) => {
+                // wrap validator function to the current call context
+                const throwIf = ({ when, error, extraArgsSupplier }) => throwMutationSpecificErrorIf(
```

Comment:
it's a new style of errors. Probably, you trying to reuse errors for another mutation. It's non explicit

Reply:
I explicitly create ERROR's for registerMultiPaymentForOneReceipt mutation.
I would like to keep throwIf function - it is just a wrapper that simplify common way to check & throw errors and. In case if you prefer to remove this wrapper - let me know